package.json

```diff
@@ -32,40 +32,42 @@
     "test:full": "npm run test && npm run cypress:test"
   },
   "devDependencies": {
+    "autoprefixer": "^9.0.1",
```

Comment:
It seems it isn't used anywhere, right?

Reply:
Well, depends on what we mean by 'used'.  I am not certain we are using the functionality, so maybe we can remove it.  But we are using it as a plugin, so …

See https://github.com/concord-consortium/portal-report/blob/master/postcss.config.js

```
module.exports = {
  plugins: [
    require('autoprefixer')({ /* ...options */ })
  ]
}

```

I would be up for removing this completely, but I was trying to just clean up the deps enough to get started on work.

Reply:
Thanks, I've looked only at webpack config and missed this one.